src/c++/perf_analyzer/concurrency_manager.h

```diff
@@ -95,14 +95,23 @@ class ConcurrencyManager : public LoadManager {
       std::shared_ptr<ThreadStat>,
       std::shared_ptr<ConcurrencyWorker::ThreadConfig>);
 
- private:
   ConcurrencyManager(
       const bool async, const bool streaming, const int32_t batch_size,
       const size_t max_threads, const size_t max_concurrency,
       const SharedMemoryType shared_memory_type, const size_t output_shm_size,
       const std::shared_ptr<ModelParser>& parser,
       const std::shared_ptr<cb::ClientBackendFactory>& factory);
 
+  // The number of worker threads with non-zero concurrencies
+  size_t active_threads_;
+
+  bool execute_;
```

Comment:
I know this is just a copy/paste, but execute_ is a poorly named variable. Not sure what this variable does,but it should be something like: "can_execute_" or "has_already_executed_"

Reply:
The only reason this showed up in the PR is because I moved it from a private access section to a protected access section. Let's refactor it in another PR.